test/pkg/repository/teardown.go

```diff
@@ -10,7 +10,10 @@ import (
 )
 
 func NSTearDown(ctx context.Context, t *testing.T, runcnx *params.Run, targetNS string) {
+	runcnx.Clients.Log.Infof("Deleting Repository in %s", targetNS)
+	err := runcnx.Clients.PipelineAsCode.PipelinesascodeV1alpha1().Repositories(targetNS).DeleteCollection(ctx, metav1.DeleteOptions{}, metav1.ListOptions{})
+	assert.NilError(t, err)
 	runcnx.Clients.Log.Infof("Deleting NS %s", targetNS)
-	err := runcnx.Clients.Kube.CoreV1().Namespaces().Delete(ctx, targetNS, metav1.DeleteOptions{})
```

Comment:
is this code never ran? i think go should have bugged us out on this?

Reply:
it is ran.. but we are deleting repo before deleting ns 
test were failing as repo were not getting deleted from etcd
when ns was deleted

Reply:
i mean the (old) code would get "no new variables on left sides"  since we were redefining twice err 

(i had to double check since it's early morning here :D 
<img width="675" alt="image" src="https://user-images.githubusercontent.com/98980/171093400-d33b8ce5-8220-4db2-955c-2d5c97ce80d7.png">
)